content/home/_directory.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="homepage-feature blocks">
+  <div class="blocks__directory">
+    <div class="blocks__icon">
+      <%= image_pack_tag "icon-school-black.svg", alt: "mortarboard icon" %>
+    </div>
+
+    <h3>Life as a teacher</h3>
+
+    <ul>
+      <li><a href="/stories">Read teachers' stories</a></li>
+      <li><a href="https://schoolexperience.education.gov.uk/">Experience a real school</a></li>
+      <li><a href="/salaries-and-benefits">Teachers' salaries</a></li>
+    </ul>
+  </div>
+  <div class="blocks__directory">
+    <div class="blocks__icon">
+      <%= image_pack_tag "icon-doc-black.svg", alt: "icon containing learning materials" %>
+    </div>
+
+    <h3>Your training</h3>
+
+    <ul>
+      <li><a href="/steps-to-become-a-teacher">How to become a teacher</a></li>
+      <li><a href="/ways-to-train">Explore ways to train</a></li>
+      <li><a href="/funding-your-training">Find your funding options</a></li>
+      <li><a href="/returning-to-teaching">Returning to teaching</a></li>
+    </ul>
+  </div>
+  <div class="blocks__directory">
+    <div class="blocks__icon">
+      <%= image_pack_tag "icon-git-black.svg", alt: "site logo icon checkmark" %>
+    </div>
+
+    <h3>Speak to teachers</h3>
+
+    <ul>
+      <li><a href="/tta-service">Get an adviser</a></li>
+      <li><a href="/events">Go to an event</a></li>
+      <li><a href="/mailinglist/signup">Sign up for emails</a></li>
+      <li><a href="#">Get school experience</a></li>
```

Comment:
@MylesJarvis this looks like it's a duplicate of the 'Experience a real school' one above

Reply:
Yep - I've removed it to bring focus to transactions.